### ISSUE TITLE
[RFE] Make it possible to use different Unix socket path, use it for distinguishing domains

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#2442]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/2438
- Created at 2014-09-11 09:59:37 by [adelton](https://pagure.io/user/adelton)
- Closed at 2020-03-24 14:07:25 as wontfix
- Assigned to nobody
---

Currently, the nss module does not allow to change the Unix socket. However, if sssd supported multiple sockets, they could be mapped to various containers (and in the container they would show up in the default location), and sssd could start making some access or namespace (domain) decisions based on the socket the client is connecting through.

### Comments

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2014-09-16 10:53:07](https://pagure.io/SSSD/sssd/issue/2438#220316)**

This should be considered in the context of #2151

milestone: NEEDS_TRIAGE => SSSD 1.13 beta

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2014-09-16 10:53:58](https://pagure.io/SSSD/sssd/issue/2438#220317)**

Fields changed

blocking:  => 2151

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2014-09-26 18:47:17](https://pagure.io/SSSD/sssd/issue/2438#220318)**

Fields changed

mark:  => 1

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2014-11-05 14:13:57](https://pagure.io/SSSD/sssd/issue/2438#220319)**

Fields changed

rhbz:  => 0

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2015-02-10 16:11:07](https://pagure.io/SSSD/sssd/issue/2438#220320)**

We can revive this ticket when we think about supporting the containerized use-case again.

milestone: SSSD 1.13 beta => SSSD 1.13 backlog
priority: major => minor

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2015-02-12 20:27:43](https://pagure.io/SSSD/sssd/issue/2438#220321)**

Mass-moving tickets not planned for the next two releases.

Please reply with a comment if you disagree about the move..

milestone: SSSD 1.13 backlog => SSSD 1.15 beta

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2017-02-24 14:21:43](https://pagure.io/SSSD/sssd/issue/2438#220322)**

**Metadata Update from @adelton**:
- Issue marked as blocked by: #2151
- Issue set to the milestone: SSSD Future releases (no date set yet)

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2020-03-11 15:31:29](https://pagure.io/SSSD/sssd/issue/2438#632502)**

**Metadata Update from @thalman**:
- Custom field blocking reset (from 2151)
- Custom field design_review reset (from 0)
- Custom field mark adjusted to on (was: 1)
- Custom field patch reset (from 0)
- Custom field review reset (from 0)
- Custom field testsupdated reset (from 0)
- Issue close_status updated to: None
- Issue tagged with: Canditate to close

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2020-03-24 14:07:24](https://pagure.io/SSSD/sssd/issue/2438#636575)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2020-03-24 14:07:25](https://pagure.io/SSSD/sssd/issue/2438#636576)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
